0000-template-protocol.md

```diff
@@ -104,6 +104,22 @@ case-sensitively.
 State management in protocols is a deep topic. For more information, please
 see [State Details and State Machines](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0003-protocols/state-details.md).
 
+### Goal Codes
+
+If your protocol defines any [goal codes](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0519-goal-codes/README.md), or if it uses goal codes defined elsewhere, and you can enumerate them, define or link them here. Use markup as shown in the sample block below.
```

Comment:
nit: use relative link

Reply:
The reason this is not a relative link is that the link has to be valid from its original position at the root of the repo, as well as from its new position inside the /features/X or /concepts/X folder.